### ISSUE TITLE
Secondary and light gray colors have accessibility issues with white

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#374]
In order to make it easy for user to see all the UI on the site, the colors should comply with section 508 contrast constraints.

- [ ] The secondary red background with white text or vice versa must be made section 508 compliant.
- [ ] Light gray on a white background or vice versa must be made 508 compliant.

**Comment by msecret** [comment]
> 
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.82:1. Recommendation: change background to #d83d2d.

![screen shot 2016-06-16 at 2 05 22 pm](https://cloud.githubusercontent.com/assets/1701077/16133168/50e60c1c-33cb-11e6-8f66-6c2f37093ed5.png)

**Comment by msecret** [comment]
>
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.94:1. Recommendation: change text colour to #767676.


![screen shot 2016-06-16 at 11 28 21 am](https://cloud.githubusercontent.com/assets/1701077/16128364/4f1439b0-33b5-11e6-8867-07f6e0782552.png)

![screen shot 2016-06-16 at 2 07 33 pm](https://cloud.githubusercontent.com/assets/1701077/16133233/9118d24c-33cb-11e6-9e7c-7f47bac3f373.png)

**Comment by msecret** [comment]
If possible, design should weigh in here.

**Comment by thisisdano** [comment]
I see the issue in the tab nav, but I don't think I see it in the `delete instance` button.

**Comment by msecret** [comment]
@thisisdano I'm using HTML codesniffer, are you using a tool to check the colors? http://squizlabs.github.io/HTML_CodeSniffer/

**Comment by thisisdano** [comment]
I used this: http://webaim.org/resources/contrastchecker/

**Comment by msecret** [comment]
For `#b23829` I'm getting a AAA fail for normal text.

**Comment by thisisdano** [comment]
Are we checking AAA or AA? Even so, the original error claims a contrast level of `3.82:1`, but I see a ratio of `6.02:1` with `#ffffff` on `#b23829`

**Comment by msecret** [comment]
yea that's strage

**Comment by thisisdano** [comment]
Tab nav contrast issue addressed with https://github.com/18F/cg-style/pull/97

**Comment by msecret** [comment]
I'm talking in #g-accessibility right now. There's a chance I used AAA when we're supposed to use AA. I also asked about the different ratio values. But yea, generally it looks like the red is fine, thanks for double checking.

**Comment by thisisdano** [comment]
Excellent — thanks for confirming!
